HerPortal/Controllers/ErrorController.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using System;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
 namespace HerPortal.Controllers;
 
+[Authorize]
```

Comment:
Can we make it so that authorisation is required by default, and we specifically allow anonymous access where it is needed? e.g. https://jasonwatmore.com/post/2022/11/28/net-6-apply-authorize-attribute-to-all-controllers